tests/kuttl/test-suites/default/test/01-assert.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: watcher.openstack.org/v1beta1
+kind: Watcher
+metadata:
+  finalizers:
+  - openstack.org/watcher
+  name: watcher-kuttl
+  namespace: watcher-kuttl-default
+status:
```

Comment:
once we have a defaulting webhook we would want to assert some of our default are set in the spec section

i.e. the default name of the database to use i.e. openstack.